modules/appeals_api/EVSS_CLAIMS.yml

```diff
@@ -0,0 +1,299 @@
+        - in: header
+          name: X-VA-Birth-Date
+          requried: true
+          schema:
+            type: string
+          description: Birth date for Veteran in iso8601 format
+        - in: header
+          name: X-VA-EDIPI
+          requried: true
+          schema:
+            type: string
+          description: Veteran's EIDPI number
+      responses:
+        '200':
+          description: Appeals retrieved successfully
+          content:
+            application/json:
+              schema:
+                required:
+                  - data
+                properties:
+                  data:
+                    $ref: "#/components/schemas/Claim"
+        '400':
+          description: Missing header
+        '401':
+          description: VAAFI authentication failure
+        '403':
+          description: Bad API Token
+        '404':
+          description: No claim found for supplied evss id
+        '422':
+          description: Invalid Header
+components:
+  securitySchemes:
+    api_key:
+      type: apiKey
+      name: apikey
+      in: header 
+  schemas:
+    Claim:
+      description: |
+        Claim with some details for the given Veteran info
+      type: object
+      properties:
+        id:
+          type: string
+          description: Claim ID from EVSS
+          example: '8347210'
+        type:
+          type: string
+          example: evss_claims
+          description: Required by JSON API standard
+        attributes:
+          type: object
+          properties:
+            date_filed:
+              type: string
+              format: date
+              description: Date in YYYY-MM-DD the claim was first filed
+              example: '2018-06-04'
+            min_est_date:
+              type: string
+              format: date
+              description: Minimum Estimated Claim Completion Date
+              example: '2019-06-04'
+            max_est_date:
+              type: string
+              format: date
+              description: Maximum Estimated Claim Completion Date
+              example: '2019-09-04'
+            phase_change_date:
+              type: string
+              format: date
+              example: '2018-06-04'
+              description: Date in YYYY-MM-DD the claim's phase was changed
+            open:
+              type: boolean
+              description: Has the claim been resolved
+              example: true
+            waiver_submitted: 
+              type: boolean
+              description: Requested Decision or Waiver 5103 Submitted
+              example: false
+            documents_needed: 
+              type: boolean
+              example: false
+              description: Does the claim require additional documents to be submitted
+            development_letter_sent: 
+              type: boolean
+              description: Indicates if a Decision Notification Letter has been sent to the Claimant regarding a benefit claim
```

Comment:
@ATeal  It looks like the Decision and Development description fields are swapped.

Reply:
Good catch fixed here: https://github.com/department-of-veterans-affairs/vets-api/pull/2650/commits/33302ea2daadcb452e92241876520005b795f36e